server/events/vcs/github_client.go

```diff
@@ -190,15 +177,19 @@ func (g *GithubClient) HidePrevPlanComments(repo models.Repo, pullNum int) error
 			continue
 		}
 		// Crude filtering: The comment templates typically include the command name
-		// somewhere in the first line. It's a bit of an assumption, but seems like
-		// a reasonable one, given we've already filtered the comments by the
-		// configured Atlantis user.
-		body := strings.Split(comment.GetBody(), "\n")
-		if len(body) == 0 {
+		// somewhere in the first line or starts with the split separator text. It's
+		// a bit of an assumption, but seems like a reasonable one, given we've
+		// already filtered the comments by the configured Atlantis user.
+		body := comment.GetBody()
+		splitBody := strings.Split(comment.GetBody(), "\n")
+		if len(splitBody) == 0 {
 			continue
 		}
-		firstLine := strings.ToLower(body[0])
-		if !strings.Contains(firstLine, models.PlanCommand.String()) {
+		firstLine := strings.ToLower(splitBody[0])
+		commentFirstLineContainsCommand := strings.Contains(firstLine, models.PlanCommand.String())
+		commentStartsWithSepStartComment := strings.HasPrefix(body, common.SepStartComment)
+
+		if !(commentFirstLineContainsCommand || commentStartsWithSepStartComment) {
```

Comment:
When an apply is run, won't this hide the current plan outputs if the plan outputs span multiple comments?